airflow/contrib/operators/dataproc_operator.py

```diff
@@ -306,6 +311,11 @@ def _build_cluster_data(self):
         if self.subnetwork_uri:
             cluster_data['config']['gceClusterConfig']['subnetworkUri'] = \
                 self.subnetwork_uri
+        if self.internal_ip_only:
+            if not self.subnetwork_uri:
+                raise AirflowException("Set internal_ip_only to true is only valid"
```

Comment:
Can you also please reword this? "Set `internal_ip_only` to true only when you pass a `subnetwork_uri `  ".